### PR TITLE
fix #75714 sportstreamtv.live

### LINKED ISSUE #75714: sportstreamtv.live
### Issue URL (Ads)
 [https://sportstreamtv.live/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fsportstreamtv.live%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard for Windows v7.6 nightly 19
Browser: | Opera
AdGuard driver: | WFP
Stealth mode options: | Hide your search queries, <br>Send Do-Not-Track header, <br>Remove X-Client-Data header from HTTP requests, <br>Strip URLs from tracking parameters, <br>Self-destructing third-party cookies (180), <br>Block Push API, <br>Block Location API
DNS filtering: | server: `https://dns-unfiltered.adguard.com/dns-query`<br>filters: `User rules`
Filters: | <b>Ad Blocking:</b><br/> EasyList, <br/>Peter Lowe's Blocklist, <br/>AdGuard Mobile Ads, <br/>AdGuard Base<br/><br/><b>Privacy:</b><br/> EasyPrivacy, <br/>AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> Adblock Warning Removal List, <br/>I don't care about cookies, <br/>EasyList Cookie List, <br/>AdGuard Annoyances<br/><br/><b>Security:</b><br/> Online Malicious URL Blocklist, <br/>Spam404, <br/>NoCoin Filter List<br/><br/><b>Language-specific:</b><br/> AdGuard Turkish
Custom filter subscriptions: | https://adguardteam.github.io/AdGuardSDNSFilter/Filters/filter.txt

### PR DESCRIPTION
**Pull request by wout-codes**
#75714

### CODE REVIEW

File: EnglishFilter/sections/general_extensions.txt [hunk 1]
```diff
@@ -157,21 +157,21 @@ imgdawgknuttz.com,getlec.blogspot.com#%#//scriptlet("abort-on-property-read", "z
...
...nkshorts.me,fusionmovies.to,lefturl.com,gamepciso.com,100count.net,gocoo.co,hdss.video,playerbiasa.xyz,akwams.*,freelitecoin.vip,freebitcoin.vip,bigb0ss.net,racaty.net,videobin.co,bdupload.asia,indishare.org,gdtot.com,livestreaming24.eu,ilinks.in,hdmovieplus.site,egy.best,egybest.*,upvid.net,filmvilag.me,bitfly.io,revivelink.com,owllink.net,goo-2o.com,akwam.*,multicanais.com,upstream.to,streamingcommunity.to,easyload.io,clipconverter.cc,readm.org,notube.net,gogo-stream.com,animesonline.cz,doctor-groups.com,kissanime.nz,gowatchseries.ac,aatfal.com,ouo.io,budcenter.site,ustreamix.to,ffmovies.co,animekisa.tv,shrinkbtc.cc,wlnk.ec,zone-annuaire.best,repackov.com,dood.*,youdbox.com,databasegdriveplayer.xyz,putlocker.style,watchmovies7.com.pk,manhwa18.com,swatchseries.to,mangatx.com,ouofly.com,playtube.ws,iir.ai,bolssc.com,linkjust1.com,fx4vip.com,mangazuki.online,elsfile.org,clicknupload.co,cima4u.io,mcloud.to,fmovies.to,aflamgulf.com,gateaflam.com,exee.io,anonymous-links.com,mediumarticles.com,pulsemens.com,aparat.cam,doramasmp4.com,jkanime.net,pcgamez-download.com,flvto.biz,movs4u.live#%#//scriptlet("abort-current-inline-script", "String.fromCharCode", "/emZnbG9hZGVk|break;case \$\./")
```
Comment (zloyden) [inline EnglishFilter/sections/general_extensions.txt:162]: `batmanstream.*` is `*` necessary here? Please provide real examples.

Comment (wout-codes) [reply EnglishFilter/sections/general_extensions.txt:162]: I don't do any thing without any purpose. I know why I added the asterisk.

Comment (wout-codes) [reply EnglishFilter/sections/general_extensions.txt:162]: @zloyden I found these two: `batmanstream.online` and `batmanstream.info`

Comment (zloyden) [reply EnglishFilter/sections/general_extensions.txt:162]: 2 domains isn't enough, but in this case ok.

**Review by zloyden — APPROVED** [review]

**Pull request merged**